package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "elemental-docs",
```

Comment:
I think this name can be changed too.

Reply:
This is the package file for the docs part only, the one for Cypress has been moved in `tests` directory. So the `name` field is good.
This file can be removed in another PR when the docs part will be fully moved in a separate repository.